tests/integrations/test_wandb.py

```diff
@@ -64,19 +63,34 @@ def test_slow(dataset_name, model_name, request):
 
 
 def _to_wandb(model, dataset):
+    run = wandb.init(project="tests")
+
     # verify that the logging of a dataset works
-    dataset.to_wandb()
+    dataset.to_wandb(run)
 
     # verify that the logging of scan results works
     scan_results = scan(model, dataset)
-    scan_results.to_wandb()
+    scan_results.to_wandb(run)
 
     # verify that the logging of test suite results works
     test_suite_results = scan_results.generate_test_suite().run()
-    test_suite_results.to_wandb()
+    test_suite_results.to_wandb(run)
 
     # Verify that the logging of the SHAP explanation charts works.
     explanation_results = explain_with_shap(model, dataset)
-    explanation_results.to_wandb()
+    explanation_results.to_wandb(run)
+
+    assert re.match("^[0-9a-z]{8}$", str(run.id))
```

Comment:
Why do you check the format of `run.id`? This is set by `wandb`, right?

Reply:
I don't remember to be honest. Yes It's a bit of a moot check.